drawOnePage.js

```diff
@@ -0,0 +1,252 @@
+function drawPageLogIn () {
```

Comment:
TO THE WHOLE FILE:
To make codestyle more accurate maybe const definitions should be set at the beginning of a function (if they do not depend on each other)
And then operations should be defined (...append, set, etc.)